src/plugins/intel_gpu/src/kernel_selector/jitter.cpp

```diff
@@ -1489,6 +1489,39 @@ JitConstants MakeTypeJitConstants(Datatype dataType, const std::string& macroNam
             type_size = "0.5f";
             is_fp = false;
             break;
+        case Datatype::BF16:
+            std::cout << " macroName " << macroName << std::endl;
+            if ( macroName == "INPUT0" || macroName == "INPUT1" || macroName == "INPUT2" || macroName == "INPUT_REORDER" ) {
```

Comment:
Why do we need this hack and other `macroName == "INPUT_REORDER"`?

Reply:
deleted